site/class-1-play/maridjuice.svg

```diff
@@ -0,0 +1,48 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 800">
```

Comment:
Note: You’re also re-adding you play assignment. Maybe remove your fork and fork again when this is merged?